daemon/algod/api/server/v2/test/handlers_test.go

```diff
@@ -806,43 +819,30 @@ func TestPostSimulateTransaction(t *testing.T) {
 	}
 }
 
-func copyInnerTxnGroupIDs(t *testing.T, dst, src *model.PendingTransactionResponse) {
+func copyInnerTxnGroupIDs(t *testing.T, dst, src *v2.PreEncodedTxInfo) {
 	t.Helper()
 
-	// msgpack decodes to map[interface{}]interface{} while JSON decodes to map[string]interface{}
-	txn := dst.Txn["txn"]
-	switch dstTxnMap := txn.(type) {
-	case map[string]interface{}:
-		srcTxnMap := src.Txn["txn"].(map[string]interface{})
-		groupID, hasGroupID := srcTxnMap["grp"]
-		if hasGroupID {
-			dstTxnMap["grp"] = groupID
-		}
-	case map[interface{}]interface{}:
-		srcTxnMap := src.Txn["txn"].(map[interface{}]interface{})
-		groupID, hasGroupID := srcTxnMap["grp"]
-		if hasGroupID {
-			dstTxnMap["grp"] = groupID
-		}
+	if !src.Txn.Txn.Group.IsZero() {
+		dst.Txn.Txn.Group = src.Txn.Txn.Group
 	}
 
-	if dst.InnerTxns == nil || src.InnerTxns == nil {
+	if dst.Inners == nil || src.Inners == nil {
```

Comment:
you renamed them because its cleaner?  or are we talking navels.

Reply:
This is because I switched from using the generated `model.SimulateResponse` to `v2.PreEncodedSimulateResponse`. The fields have slightly different names in go but they encode identically